sdk/nodejs/runtime/main.go

```diff
@@ -28,11 +28,21 @@ func (t *TypescriptSdk) ModuleRuntime(ctx context.Context, modSource *Directory,
 		return nil, err
 	}
 
+	ctr = ctr.WithDirectory(genDir, ctr.Directory(sdkSrc), ContainerWithDirectoryOpts{
+		Exclude: []string{
+			"node_modules",
+			"dist",
+			"codegen",
+			"**/test",
+			"runtime",
+		},
+	})
+
```

Comment:
You already have this in `Codegen`, why not move to `CodegenBase`?

Reply:
Because it's actually not included in the codegen directory, I don't understand why though

Reply:
Can you paste a diff of how you tried to do that?

Reply:
Fixed in https://github.com/dagger/dagger/pull/6270/commits/691d72fb580b718d93cc5f1dcc88ce46e7d1389f